index.js

```diff
@@ -58,6 +58,31 @@ function onInitHooks() {
 	)
 }
 
+function onInitCreateAdmin() {
+	const email = process.env.BALENA_ROOT_EMAIL || '';
+	const password = process.env.BALENA_ROOT_PASSWORD || '';
+
+	if (email == '' || password == '') {
+		return;
+	}
+
+	console.log("Creating admin account...")
+
+	const { runInTransaction } = require('./src/platform');
```

Comment:
These are untyped, you can either use
```typescript
import('./src/platform')
.then(({ runInTransaction }) => {
	...
}
```
 or
```typescript
import * as _platform from './src/platform'
...
const { runInTransaction }: typeof _platform = require('./src/platform')
```
to make sure they get typed.  You can also use:
```typescript
const { runInTransaction } = await import('./src/platform')
```
as a variation of the first one